client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/index.tsx

```diff
@@ -80,6 +80,7 @@ const PatternAssembler = ( props: StepProps & NoticesProps ) => {
 	const [ sectionPosition, setSectionPosition ] = useState< number | null >( null );
 	const wrapperRef = useRef< HTMLDivElement | null >( null );
 	const [ activePosition, setActivePosition ] = useState( -1 );
+	const [ surveyDismissed, setSurveyDismissed ] = useState( false );
```

Comment:
![badge](https://img.shields.io/badge/review-nits-green.svg)
I'm wondering whether it's necessary to store (to localStorage or somewhere) whether a user has dismissed the survey after they leave the page.

Reply:
I think it's fine for now since it's not very intrusive. We can consider this in the future 🙂